sample-functions/gif-maker/entry.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
+
```

Comment:
Was this actually picked up by your linter?

Reply:
I run scripts to make sure there is a blankline after #!/bin/sh and only 1 at the end. That may have been run into this.